pulumi/grapl/__main__.py

```diff
@@ -136,11 +141,19 @@ def main() -> None:
         kafka = Kafka("kafka")
 
         job_vars = pulumi.Output.all(
+            analyzer_bucket=analyzers_bucket.bucket,
+            # This should be dispatched-analyzer-bucket, which doesn't appear to exist?
+            analyzer_dispatched_bucket=analyzers_bucket.bucket,
```

Comment:
I found it - I was as confused as you were until I realized it was doing string interpolation to construct "dispatched-analyzer-bucket"
```
    dispatched_analyzer_emitter = emitter.EventEmitter("dispatched-analyzer")
````
and in EventEmitter:
```
        bucket_name = f"{event_name}-bucket"
        pulumi.export(bucket_name, self.bucket.bucket)
```

so you probably want `dispatched_analyzer_emitter.bucket.bucket` ?

Reply:
Ah yes. I think I actually did this with a different bucket too 🤦 . Fixing now